src/Apps/__test__/Fixtures/Order.ts

```diff
@@ -86,8 +89,15 @@ export const UntouchedOrder = {
   },
 }
 
+export const UntouchedOfferOrder = {
+  ...UntouchedBuyOrder,
+  mode: "OFFER",
+  state: "PENDING",
+  taxTotal: null,
```

Comment:
#superminor `taxTotal` and `state` can be omitted here? Seems they have the same values in `UntouchedBuyOrder`.